src/main/java/com/codeborne/selenide/Driver.java

```diff
@@ -95,4 +98,17 @@ default SelenideTargetLocator switchTo() {
   default Actions actions() {
     return new Actions(getWebDriver());
   }
+
+  @CheckReturnValue
+  @Nullable
+  default SessionId getSessionId() {
+    WebDriver driver = getWebDriver();
+    if (getWebDriver() instanceof EventFiringWebDriver) {
+      driver = ((EventFiringWebDriver) getWebDriver()).getWrappedDriver();
+    }
+    if (driver instanceof RemoteWebDriver) {
+      return ((RemoteWebDriver) driver).getSessionId();
+    }
+    return null;
```

Comment:
As I know all drivers implements `RemoteWebDriver` interface, so instead of return `null` it should always return `((RemoteWebDriver) driver).getSessionId()`, isn't it?

Reply:
I mean it is impossible by webdriver protocol to have driver implementations that don't have `sesisonId`

Reply:
Yes you are right. fixed